.github/runtime_specs/westend.json

```diff
@@ -0,0 +1,17 @@
+{
+    "pallets": {
+      "1": {
+        "constants": {
+          "EpochDuration": {
+            "value": [ 88, 2, 0, 0, 0, 0, 0, 0 ]}
+        }
+      },
+
+      "2": {
+        "constants": {
+          "MinimumPeriod": {
```

Comment:
Does it detect typos in the constant names?

Reply:
If the spec contains:
```
 "2": {
        "constants": {
          "foo": { ... }
```
 
and the metadata :
```
 "2": {
        "constants": {
          "foobar": { ... }
```

It will not detect a typo but it will not find `foo` and fail.